src/main/java/com/taskadapter/redmineapi/bean/TimeEntry.java

```diff
@@ -139,13 +143,25 @@ public String getUserName() {
     public void setUserName(String userName) {
         storage.set(USER_NAME, userName);
     }
+    
+    public CustomField getCustomField(String name) {
+    		return 	((List<CustomField>)storage.get(CUSTOM_FIELDS)).stream().filter(a -> a.getName().equals(name)).findFirst().get();
```

Comment:
I wonder if there was a particular reason to use a non-generified Property<List> for CUSTOM_FIELDS field?
see 
`public final static Property<Set<CustomField>> CUSTOM_FIELDS = (Property<Set<CustomField>>) new Property(Set.class, "customFields");`
in Issue class for comparison.
this allows to avoid type casting later, like this:
` for (CustomField customField : storage.get(CUSTOM_FIELDS)) {`

Reply:
findFirst().get() will produce an exception if nothing is found. this is inconsistent with the way custom fields are resolved in Issue class.
a better way would be to return Optional itself. but at least need to be consistent with the other classes.